test/new/redos-nolink.md

```diff
@@ -0,0 +1 @@
+![\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]\[[]!\
```

Comment:
why is redos-newlink.html empty?

Reply:
What would be expected otherwise? Think in this case we're not so much worried that the write HTML gets spit out and more concerned on whether the test can finish - given that Travis died...literally on it, I'd say it's an effective test. :)

Reply:
I figured that was to hold the expected output. I don't know what the expected output is, just so long as it gets generated without REDOS.